src/app/demo/demo.module.ts

```diff
@@ -47,6 +47,12 @@ export const routes = [
     description: 'This is a place for experiments, dev only'
   },
   {
+    path: 'feedback-page',
+    loadChildren: './feedback-page/feedback-page.module#FeedbackPageModule',
+    name: 'Feedback',
```

Comment:
please get rid of both name and description lines (So that it doesn't get displayed in the index)